internal/client/client.go

```diff
@@ -124,31 +161,6 @@ func (c *client) UpdatePipeline(pipeline *Pipeline) (*Pipeline, error) {
 	return updated, nil
 }
 
-func readBody(resp *http.Response, err error) error {
```

Comment:
This was just moved to the top.  It was annoying having it below interface methods